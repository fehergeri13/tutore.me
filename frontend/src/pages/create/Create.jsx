import React from 'react';
import "./create.less"
import Header from "../../components/header/Header";

export default class Create extends React.Component {
    render() {
        return <div className="create-page">
            <div className="creator">
                <h2>Új hirdetés felvétele</h2>

                <form>
                    <label className="title"><span>Hirdetés neve</span><input type="text"/></label>
                    <label className="text"><span>Hirdetés szövege</span>
                        <textarea defaultValue="Valami szöveg"/>
                    </label>

                    <fieldset className="type">
                        <legend>Típus</legend>
                        <label><input type="radio" name="contact" value="keres"/><span>Keres</span></label>
                        <label><input type="radio" name="contact" value="keres"/><span>Kínál</span></label>
                    </fieldset>

                    <fieldset className="subject">
                        <legend>Tantárgy</legend>
                        <label><input type="checkbox" name="contact" value="keres"/><span>Informatika</span></label>
                        <label><input type="checkbox" name="contact" value="keres"/><span>Matematika</span></label>
                        <label><input type="checkbox" name="contact" value="keres"/><span>Fizika</span></label>
                        <label><input type="checkbox" name="contact" value="keres"/><span>Kémia</span></label>
                    </fieldset>

                    <div className="buttons">
                        <button>Törlés</button>
                        <button>Mentés</button>
                    </div>
                </form>
            </div>

        </div>;
    }
}
