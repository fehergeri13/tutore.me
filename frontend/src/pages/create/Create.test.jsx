import React from 'react';
import renderer from 'react-test-renderer';
import Create from "./Create";

describe('Create', () => {
    it('renders correctly', () => {
        const tree = renderer.create(
            <Create />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
