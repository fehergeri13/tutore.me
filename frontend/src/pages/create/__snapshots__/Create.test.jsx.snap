// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create renders correctly 1`] = `
<div
  className="create-page"
>
  <div
    className="header"
  >
    <h1>
      tutore.me
    </h1>
    <div
      className="nav"
    >
      <ul
        className="links"
      >
        <li>
          <a
            href="#"
          >
            Hirdetések
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Új hirdetés
          </a>
        </li>
      </ul>
    </div>
    <div
      className="user"
    >
      <ul
        className="links"
      >
        <li>
          <a
            href="#"
          >
            Üzenetek
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Profilom
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Kilépés
          </a>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="creator"
  >
    <h2>
      Új hirdetés felvétele
    </h2>
    <form>
      <label
        className="title"
      >
        <span>
          Hirdetés neve
        </span>
        <input
          type="text"
        />
      </label>
      <label
        className="text"
      >
        <span>
          Hirdetés szövege
        </span>
        <textarea
          defaultValue="Valami szöveg"
        />
      </label>
      <fieldset
        className="type"
      >
        <legend>
          Típus
        </legend>
        <label>
          <input
            name="contact"
            type="radio"
            value="keres"
          />
          <span>
            Keres
          </span>
        </label>
        <label>
          <input
            name="contact"
            type="radio"
            value="keres"
          />
          <span>
            Kínál
          </span>
        </label>
      </fieldset>
      <fieldset
        className="subject"
      >
        <legend>
          Tantárgy
        </legend>
        <label>
          <input
            name="contact"
            type="checkbox"
            value="keres"
          />
          <span>
            Informatika
          </span>
        </label>
        <label>
          <input
            name="contact"
            type="checkbox"
            value="keres"
          />
          <span>
            Matematika
          </span>
        </label>
        <label>
          <input
            name="contact"
            type="checkbox"
            value="keres"
          />
          <span>
            Fizika
          </span>
        </label>
        <label>
          <input
            name="contact"
            type="checkbox"
            value="keres"
          />
          <span>
            Kémia
          </span>
        </label>
      </fieldset>
      <div
        className="buttons"
      >
        <button>
          Törlés
        </button>
        <button>
          Mentés
        </button>
      </div>
    </form>
  </div>
</div>
`;
